ontobio/io/gafparser.py

```diff
@@ -144,13 +156,16 @@ def parse_line(self, line):
                     else:
                         logger.info("Detected GAF version {}, so using 2.1".format(version))
                         self.version = self.default_version
+                        # Compute the cell component subclass closure
```

Comment:
@dougli1sqrd Does this here mean the closure will be recalculated on every call of `parse_line` for a 2.1 GAF?

Reply:
No, this only computes the closure if we are in the headers and once we have detected that we are in gaf 2.1. So this should happen only once per gaf 2.1 file.

Reply:
Ah, right. I just realized that after staring harder. All is good, thanks!